gen-prometheus.py

```diff
@@ -0,0 +1,7 @@
+from sys import argv
+from string import split
+ips=split(argv[1],",")
+port=argv[2]
+print "- targets:"
+for addr in map(lambda x: x + ":" + port, ips):
```

Comment:
You need a python header comment (either python or python3)
For what you're doing here, you can just add bash loop (like the one that handle versions in start-all.sh)

If you prefer to keep the python, just add the header and I'll merge it